runtime/compiler/control/J9Options.cpp

```diff
@@ -1916,6 +1916,8 @@ J9::Options::fePreProcess(void * base)
             }
          }
       }
+   if (TR::Compiler->target.cpu.isZ())
```

Comment:
Don't you mean `if (!TR::Compiler->target.cpu.isZ())` ?

Reply:
Fixed in https://github.com/eclipse/openj9/pull/5926/commits/25d93bbe10b65fad9f9afc23c0ad8386e77b112d